### PR TITLE
Added numerous new sources + Fixed some formatting and grammar mistakes

### PR DESCRIPTION
**Pull request by JoshuaKimsey**
Added: Multiple new sources of information, including guides, videos, a new img2img tool, and an upcoming plugin for Krita.

Fixed: A few formatting and grammar issues in the readme and contributing files.

Definitely will add more as time moves along! :)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -65,11 +67,13 @@ Models, commercial and open source, that you can either try for free and pay for
 ![](img2img.png)
 > img2img refers to the practice of feeding a model an init image AND a prompt. 
 - [stable-img-to-img](https://replicate.com/afiaka87/stable-image-to-image) - Hosted version of img2img tool from stable diffusion. Provide an image + a prompt
+- [Diffuse-The-Rest](https://huggingface.co/spaces/huggingface/diffuse-the-rest) - A simple Hugging Face img2img tool that allows for image upload and/or simplistic drawing with a prompt
```
Comment (altryne) [inline readme.md:70]: 🔥

File: readme.md [hunk 2]
```diff
@@ -65,11 +67,13 @@ Models, commercial and open source, that you can either try for free and pay for
...
 
 ## Prompt engineering
-The emerging field of the Prompt Engineering is just beginning, and yet there are tools released for us every day. Follow this repo for the best tools as they become "Awesome"
+> The emerging field of the Prompt Engineering is just beginning, and yet there are tools released for us every day. Follow this repo for the best tools as they become "Awesome"
 ### Tools for prompt engineers
-  - [Lexica](https://lexica.art/) - A prompts search engine
+  - [Lexica](https://lexica.art/) - A Stable Diffusion prompts search engine
+  - [Libraire](https://libraire.ai/) - Another Stable Diffusion prompts search engine, with over 10M images and prompts
```
Comment (altryne) [inline readme.md:76]: 🔥

**Review by altryne — APPROVED** [review]
Thank you kindly friend!

File: readme.md (hunk 1 above)
Comment (JoshuaKimsey) [reply readme.md:70]: IKR? I wanna see something simple like this added to the img2img in the WebUI. Can be super simple, and give people this ability easily.

File: readme.md (hunk 2 above)
Comment (JoshuaKimsey) [reply readme.md:76]: Both do tend to overlap, but currently Libraire has a larger database, plus there's something I like about searching on it, even though the UI isn't quite as nice as Lexica. Both are 10/10 resources though!

**Pull request merged**